lib/src/configs/cosim_wrap_config.dart

```diff
@@ -107,6 +107,8 @@ class CosimWrapConfig extends CosimProcessConfig {
       Cosim.registrees,
       dumpWavesString: dumpWavesString,
     );
+    _createRequirementsTxt(directory: directory);
+    _pythonVEnvProcess(directory: directory);
```

Comment:
I think we should make creation and usage of the virtual environment optional, so that if someone has set up their own already they aren't forced to use this?